### PR TITLE
Add `compare()` function to `selector-specificity`

### PR DESCRIPTION
**Pull request by ybiquitous**
This PR adds a `compare()` utility function like the [`specificty`](https://www.npmjs.com/package/specificity) package or the [`@bramus/specificity`](https://www.npmjs.com/package/@bramus/specificity) have.

Also, Stylelint may need it. See https://github.com/stylelint/stylelint/pull/6131

### CODE REVIEW

File: packages/selector-specificity/test/_require.cjs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 const assert = require('assert');
-const selectorSpecificity = require('@csstools/selector-specificity');
+const { default: selectorSpecificity, compare } = require('@csstools/selector-specificity');
```
[the remark this answers is not in the document]
Comment (romainmenke) [reply packages/selector-specificity/test/_require.cjs:2]: I don't mind doing the needed changes after this PR

[the remark this answers is not in the document]
Comment (romainmenke) [reply packages/selector-specificity/test/_require.cjs]: update : I've made the needed changes 😄

File: packages/selector-specificity/README.md [hunk 2]
```diff
@@ -27,6 +27,22 @@ console.log(specificity.c); // 0
 _`selectorSpecificity` takes a single selector, not a list of selectors (not : `a, b, c`).
 To compare or otherwise manipulate lists of selectors you need to call `selectorSpecificity` on each part._
 
+### Comparing
+
+The package exports a utility function to compare two specificities.
+
+```js
+import selectorSpecificity, { compare } from '@csstools/selector-specificity';
```
Comment (romainmenke) [inline packages/selector-specificity/README.md:35]:
```suggestion
import { selectorSpecificity , compare } from '@csstools/selector-specificity';
```

File: packages/selector-specificity/README.md [hunk 3]
```diff
@@ -27,6 +27,22 @@ console.log(specificity.c); // 0
...
+
+const s1 = selectorSpecificity(ast1);
+const s2 = selectorSpecificity(ast2);
+compare(s1, s2); // -1, 0, 1
```
Comment (romainmenke) [inline packages/selector-specificity/README.md:39]:
```suggestion
compare(s1, s2); // -1 | 0 | 1
```

**Review by romainmenke — COMMENTED** [review]
Two final notes 🎉

**Comment by ybiquitous** [comment]
Done. Thanks for the review!

File: packages/selector-specificity/README.md [hunk 4]
```diff
@@ -27,6 +27,22 @@ console.log(specificity.c); // 0
 _`selectorSpecificity` takes a single selector, not a list of selectors (not : `a, b, c`).
 To compare or otherwise manipulate lists of selectors you need to call `selectorSpecificity` on each part._
 
+### Comparing
+
+The package exports a utility function to compare two specificities.
+
+```js
+import { selectorSpecificity , compare } from '@csstools/selector-specificity';
```
Comment (ybiquitous) [inline packages/selector-specificity/README.md:35]:
```suggestion
import { selectorSpecificity, compare } from '@csstools/selector-specificity';
```

**Comment by romainmenke** [comment]
@ybiquitous Thank you for this 🎉 
We will release this shortly!

**Pull request merged**